pokemongo_bot/cell_workers/spin_fort.py

```diff
@@ -100,8 +105,9 @@ def work(self):
                 result = c.fetchone()
                 c.execute("SELECT DISTINCT COUNT(pokestop) FROM pokestop_log WHERE dated >= datetime('now','-1 day')")
                 if c.fetchone()[0] >= self.config.get('daily_spin_limit', 2000):
-                    self.emit_event('spin_limit', formatted='WARNING! You have reached your daily spin limit')
-                    sys.exit(2)
+                   if datetime.now() >= self.next_update:
+                       self.emit_event('spin_limit', formatted='WARNING! You have reached your daily spin limit')
+                       self._compute_next_update()
```

Comment:
Running is done by MoveToFort or FollowSpiral etc., not by SpinFort. I am also not sure whether returning ERROR here would count towards softban detection... So returning SUCCESS is probably the most nonintrusive way.

Plus - sometimes, you want actually to continue running and apturing pokemons and sniping even with a reached daily limit.